develop/reference_sql.rst

```diff
@@ -70,6 +70,132 @@ As an example, if you want to run the hll_union aggregate function on your data
 
 You can then call sum(hll_column) to roll up those columns within the database. Please note that these custom aggregates need to be created both on the coordinator and the workers.
 
+.. _topn:
+
+Estimating Top N Items
+~~~~~~~~~~~~~~~~~~~~~~
+
+Similar to how HLL efficiently estimates the number of distinct items in a set, the `TopN PostgreSQL extension <https://github.com/citusdata/postgresql-topn>`_ simultaneously counts many items. It, too, stores an approximation in a bounded amount of space. It is also pre-installed on Citus Cloud.
+
+Fundamentally the extension tracks a multi-set as a JSON object.
```

Comment:
Are the below two points necessary? Are you thinking of these as being more "reference" type information for the extension?

Reply:
I wanted to clarify what the extension really does, in addition to a scenario where it's used. When I first saw the repo I wasn't totally sure. Then I experimented with it a little and saw what the basic operation is, and that it is pretty simple.